content/docs/error-boundaries.md

```diff
@@ -22,6 +22,8 @@ Error boundaries are React components that **catch JavaScript errors anywhere in
 > * Server side rendering
 > * Errors thrown in the error boundary itself (rather than its children)
 
+> In development an error overlay will be shown on top of the screen reguardless if you're using error boundaries or not, you can close the overlay by pressing esc
```

Comment:
That depends on the framework you're using (e.g nextjs or create-react-app). It is not included in React itself. So we should probably include that an error overlay might be shown depending on the framework you're using. 

I would leave out instructions on how to close it since that depends on the specific implementation of the error overlay.

Reply:
So something like this "In development an error overlay might be shown on top of the screen regardless if you're using error boundaries or not, usually you can close this overlay"

Reply:
Yeah, and I would add that this depends on the framework you're using and give some examples (create-react-app, nextjs). 

Usually I would be careful with documenting 3rd party behavior but at this point it's somewhat convention that your React framework should implement some error overlay.

Reply:
I've changed to "In development, an error overlay might be shown on top of the screen regardless if you're using error boundaries or not, create-react-app and nextjs implement this error overlay"